backend/src/main/java/gov/cdc/usds/simplereport/api/converter/FhirConverter.java

```diff
@@ -273,4 +281,66 @@ public static Extension convertToTribalAffiliationExtension(String tribalAffilia
     }
     return null;
   }
+
+  public static Device convertToDevice(DeviceType deviceType) {
+    if (deviceType != null) {
+      return convertToDevice(
+          deviceType.getManufacturer(),
+          deviceType.getModel(),
+          Objects.toString(deviceType.getInternalId(), ""));
```

Comment:
What's the reasoning behind having this as `Objects.toString()` instead of `deviceType.getInternalId().toString()`?